ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java

```diff
@@ -232,13 +234,27 @@ public void run() {
         stopwatch.stop();
 
         if (lastLoop) {
+          Transaction tx =
+              new Transaction(
+                  0,
+                  Wei.ZERO,
+                  Long.MAX_VALUE,
+                  Optional.ofNullable(receiver),
+                  Wei.ZERO,
+                  null,
+                  callData,
+                  sender,
+                  Optional.empty());
+
+          final Gas intrinsicGasCost =
+              component.getGasCalculatorAtBlock().apply(0).transactionIntrinsicGasCost(tx);
+          Gas evmGas = gas.minus(messageFrame.getRemainingGas());
           System.out.println(
```

Comment:
Why don't we use the logger present in the class to log this? `LogManager.getLogger()`

Reply:
Logger create too much extra line junk with the default setup for what should be a CLI focused tool.  However, I did change to use the printstream from the PicoCLI invocation.